README.md

```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
 ./develop_server stop
 ```
 
+## Heroku Review Apps ##
```

Comment:
Yeah I agree that it doesn't make much sense to put this up above, as this is more CI-focused. The Testing Locally fix sounds fine to me.